runtime/arc.js

```diff
@@ -231,11 +231,11 @@ ${this.activeRecipe.toString()}`;
       storageProviderFactory: manifest._storageProviderFactory,
       context
     });
-    manifest.stores.forEach(store => {
+    await Promise.all(manifest.stores.map(async store => {
       if (store.constructor.name == 'StorageStub')
-        store = store.inflate();
+        store = await store.inflate();
       arc._registerStore(store, manifest._storeTags.get(store));
```

Comment:
This will lead to reorderings (the inflating might take different lengths of time for different stores). Will that become an issue in terms of resolution orders?